apps/src/code-studio/pd/application_dashboard/quick_view.jsx

```diff
@@ -7,14 +7,39 @@
  *        /csp_facilitators
  */
 import React, {PropTypes} from 'react';
+import Select from "react-select";
+import "react-select/dist/react-select.css";
 import QuickViewTable from './quick_view_table';
 import Spinner from '../components/spinner';
-import {Button} from 'react-bootstrap';
 import $ from 'jquery';
+import {TeacherApplicationStatuses, FacilitatorApplicationStatuses} from './constants';
+import {
+  Button,
+  FormGroup,
+  ControlLabel,
+  Row,
+  Col
+} from 'react-bootstrap';
+
+// Default max height for the React-Select menu popup, as defined in the imported react-select.css,
+// is 200px for the container, and 198 for the actual menu (to accommodate 2px for the border).
+// React-Select has props for overriding these default css styles. Increase the max height here:
+const selectMenuMaxHeight = 400;
+const selectStyleProps = {
+  menuContainerStyle: {
+    maxHeight: selectMenuMaxHeight
```

Comment:
Can we just do this as "auto?" Then won't it be large enough for everything?

Reply:
ooh I will try that.

Reply:
From what I remember when I did this for the [filters](https://github.com/code-dot-org/code-dot-org/blob/e4138f164548a2ad6a89994ee2325da1bded2cbc/apps/src/code-studio/pd/workshop_dashboard/workshop_filter.jsx#L38-L37), auto didn't work for 2 reasons: we actually want to constrain it for longer lists (probably not an issue here with a fixed set of statuses), and because of the 2px border. I suppose we could wrap it up in another component that takes a single height and handles the 2px difference, now that we're using this in multiple places. It might be worth a little experimentation.